docs/system-settings/release-notes.md

```diff
@@ -1,16 +1,16 @@
 ---
 id: release-notes 
-title: Release notes
-sidebar_label: Release notes
+title: Release Notes
+sidebar_label: Release Notes
 ---
 
 The Release Notes contain details about features added, improvements,
 bugs fixed and known issues in each release of the console software.
 
 Release notes for the version of software running on your console can be
-found by clicking the Help Release Notes option on the toolbar.
+found by clicking the **Help**, then **Release Notes** option on the toolbar.
 
 The most up-to-date version of the Release Notes can be found on the
-Avolites website www.avolites.com.
+[Avolites website](//avolites.com/software).
```

Comment:
This URL looks wrong? Surely should be http://

Reply:
It'll work, it's a protocol specific link but was less to type